src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -238,6 +238,18 @@ public enum ConfigNodes {
 			"",
 			"# Maximum distance between homeblocks.",
 			"# This will force players to build close together."),
+	TOWN_NEW_TOWN_MIN_DISTANCE_FROM_TOWN_PLOT(
+			"town.new_town_min_distance_from_town_plot",
+			"-1",
+			"",
+			"# The minimum distance that a new town must be from nearby town's plots.",

```

Comment:
```suggestion
			"# The minimum distance that a new town must be from nearby towns' plots.",

```

Reply:
The other new config option has a similar thing, but the editor/suggester isn't submitting it correctly.